pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths = test
```

Comment:
Ok, that is excellent. So we can just keep it for travis but remove it everywhere else?